### PR TITLE
Feature/biodiversity glads aoi widget

### PR DESCRIPTION
**Pull request by edbrett**
## Overview

A new shiny widget to display the GLAD alerts within a country of region, broken down by sub regions, and ranked by total area or relative percentage of tree cover extent.

screen shot 2018-03-27 at 18 31 06

### CODE REVIEW

File: app/javascript/pages/country/data/weeks.json [hunk 1]
```diff
@@ -1,4 +1,8 @@
 [
+  {
+    "label": "4 weeks",
+    "value": 4
```
Comment (pjosh) [inline app/javascript/pages/country/data/weeks.json]: this affects the glad-alerts widget. Is this right?

Comment (edbrett) [reply app/javascript/pages/country/data/weeks.json]: Ah cheese 🤔 . Yes I should have put in the .env. And yes weeks selector needs updating. 🏎 💨

Comment (edbrett) [reply app/javascript/pages/country/data/weeks.json]: Fixed!

**Comment by 01painadam** [comment]
In an attempt to speed things up a little I've added a further condition in the sql which only retrieves the last years worth of glad alerts - @edbrett / @pjosh , can you double check my syntax here please :)

I also added the widget to the conservation tab.

**Comment by edbrett** [comment]
We can review Monday morning as it doesnt have to be production until the afternoon. Provisionally looks good.

**Comment by 01painadam** [comment]
A few further updates based on feedback:

- changes `alerts_count` to `alerts` in sql following table update
- Added total count in place of area% to dynamic sentence
- Fixes small bug in dynamic sentence

**Comment by 01painadam** [comment]
- Removes Glad-Biodiversity from Summary tab as requested
- Adds missing locations to GAIN widgets

**Pull request merged**